tests/acceptance/test_issue_details.py

```diff
@@ -15,6 +15,9 @@
 
 
 class IssueDetailsTest(AcceptanceTestCase, SnubaTestCase):
+    def __init__(self, features):
+        self.features = features
```

Comment:
Why is this needed now?

Reply:
I was just trying something ... I've updated this code ;-)